source/whats-new.txt

```diff
@@ -45,6 +43,82 @@ What's New in 1.12
      deprecated. You can create a client and connect in one call
      by using the ``mongo.Connect()`` method.
 
+New features of the 1.12 Go driver release include: 
+
+Queryable Encryption
+~~~~~~~~~~~~~~~~~~~~
+
+The release adds production-ready support for
+:ref:`qe-manual-feature-qe` (QE). To use
+the QE feature, you must use MongoDB Server version 7.0 or later and
+``libmongocrypt`` version 1.8.0 or later.
+  
+This release introduces the ``ClientEncryption.CreateEncryptedCollection()``
+method, which automatically creates data encryption keys when
+creating a new encrypted collection. This method adds the ability to
+fetch KMS credentials automatically from Azure, GCP, and AWS
+environments.
+
+Logging Interface
+~~~~~~~~~~~~~~~~~
+
+You can now record detailed information about connection management and
+command execution.
+
+The logging configuration adds a new ``LogSink`` interface that is
+compatible with the `logr.LogSink <https://pkg.go.dev/github.com/go-logr/logr#LogSink>`__ interface.
+You can use existing ``logr`` adapters with the logging interface
+such as :github:`zerologr <go-logr/zerologr>` or :github:`zapr <go-logr/zapr>`.
+    
+You can also implement your own ``LogSink`` adapter. The following
+example shows how to integrate logging with an existing :github:`zerolog
+<rs/zerolog>` logger:
+
+.. code-block:: go
+
+   sink := zerologr.New(&myLogger).GetSink()
+   loggerOptions := options.Logger().SetSink(sink)
+   options.Client().SetLoggerOptions(loggerOptions)
+
+Additional BSON Marshalling Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release adds new features to the ``options`` package to specify
+various BSON marshalling and unmarshalling behaviors.
```

Comment:
Question:
Is this referring to struct tag behavior? If so, perhaps it would be good to mention that in the intro sentence and/or link to https://www.mongodb.com/docs/drivers/go/current/fundamentals/bson/#struct-tags so that the reader can connect with the concept.